addins/Cake.Grunt.yml

```diff
@@ -0,0 +1,10 @@
+Name: Cake.Grunt
+NuGet: Cake.Grunt
+Assemblies:
+- "/**/Cake.Gulp.dll"
+Repository: https://github.com/cake-contrib/cake-grunt
+Author: Ken Lea
+Description: "A set of aliases for Cake to help with running Grunt scripts as part of a build. (Copied from https://github.com/cake-contrib/cake-gulp by Phil Oyston)"
+Categories:
+- Node
+s
```

Comment:
This is invalid

Reply:
Should be done now, don’t know how that character got in there 🙄